docs/api/index.rst

```diff
@@ -11,6 +11,17 @@ API
 Changelog
 ---------
 
+1.13 (2016-10-18)
+'''''''''''''''''
+
+- Add support to JSON ``patch`` format when using 
+  ``Content-Type: application/json-patch+json`` (as in RFC 6902).
+- when using JSON ``patch``, changes should be described as a list of operations
+  (e.g ``[{"op": "add", "path": "/data/my_field", "value": "my_value"}]``).
+- On JSON ``patch`` operations, permissions are trated as a dictionary with no values.
+  On these operations, the ``value`` field can be omitted.
+  (e.g ``[{"op": "add", "path": "/permissions/read/system.Everyone"}]``).
```

Comment:
I don't understand this question. I'd probably replace the second and third bullet points and just add a sentence to the first that says something like

```rst
For more details, see `JSON Patch Format <link here>`_.
```

Are you asking where the link should point to? I would have it point to the JSON Patch section above, if that's possible.